<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>473</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nonlinear Knapsack problem solver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>353</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="fileLayout">
       <item>
        <widget class="QLabel" name="fileLabel">
         <property name="text">
          <string>File :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="fileField"/>
       </item>
       <item>
        <widget class="QPushButton" name="fileButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="generalOptionsLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="timeLimitLabel">
         <property name="text">
          <string>Time limit (in seconds)</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="timeLimitField">
         <property name="maximum">
          <number>1800</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="configurationNumberLabel">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Number of Configurations</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="configurationNumberField">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>4</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="configurationChooserLabel">
       <property name="text">
        <string>Configuration number shown below:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="configurationChoosetLayout">
       <item>
        <widget class="QRadioButton" name="config1Button">
         <property name="text">
          <string>1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="config2Button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>2</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="config3Button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="config4Button">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>4</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="algorithmLabel">
         <property name="text">
          <string>Algorithm</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="algorithmField">
         <item>
          <property name="text">
           <string>B-BB: Simple Branch and Bound</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B-OA: OA Decomposition</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B-QG: Quesada and Grossmann</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>B-Hyb: Hybrid</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="rootNumberLabel">
         <property name="text">
          <string>Number resolve at root</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="rootNumberField">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="nodeNumberLabel">
         <property name="text">
          <string>Number resolve at node</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="nodeNumberField">
         <property name="maximum">
          <number>999999999</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="variableSelectionLabel">
         <property name="text">
          <string>Variable selection mode</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="variableSelectionField">
         <item>
          <property name="text">
           <string>Most Fractional</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Strong Branching</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Reliability Branching</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>QP Strong Branching</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LP Strong Branching</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>NLP Strong Branching</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Osi simple</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Osi strong</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Random</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="solveButton">
       <property name="text">
        <string>Solve problem</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
